pilot/cmd/pilot-agent/main.go

```diff
@@ -196,6 +200,31 @@ var (
 
 			log.Infof("Monitored certs: %#v", certs)
 
+			if templateFile != "" && proxyConfig.CustomConfigFile == "" {
+				opts := make(map[string]string)
```

Comment:
`map[string] interface{}` ?

so you can have `opts["ControlPlaneAuth"] = true`

Reply:
In templates, any non-nil value is treated the same. So "true" and true are the same in string templates.